### ISSUE TITLE
"<no value>" after templating

### ISSUE THREAD

**Issue by Z9n2JktHlZDmlhSvqc9X2MmL3BwQG7tk** [VictoriaMetrics/VictoriaMetrics#4012]
### Is your feature request related to a problem? Please describe

When I use label name that does not exist in the expression result I get `<no value>` label value in alert.

Test alerting rule
```yaml
groups:
  - name: test
    rules:
      - alert: qqq
        expr: 1
        labels:
          alert: problem_only
          host: '{{ $labels.host }}'
#          host_display: '{{ if ne $labels.host "" }}{{ $labels.host }}{{ else }}label host is empty{{ end }}'
          host_display: '{{ if and (ne $labels.host "") (ne $labels.host nil) }}{{ $labels.host }}{{ else }}label host is empty{{ end }}'
        annotations:
          problem: 'just a test'
```

![image](https://user-images.githubusercontent.com/22332263/227735258-b8e5701c-06a9-41f6-813a-3be916d8188d.png)

Prometheus was converting `nil` to an empty string, so I could use `if ne $labels.host ""`, but now I need to add an extra `nil` check to all affected alerting rules. It is not so convenient. And what's more, it's not so obvious to check label value against `nil` (first I tried to check `if ne $labels.host "<no value>"`).

### Describe the solution you'd like

Propose to convert non-existent label values to an empty string.
What do you think ?

### Additional information

Version: victoria-metrics-20221126-045002-tags-v1.84.0-0-gad105147d
Type: single node

**Comment by zekker6** [comment]
@hagen1778 What do you think about this?
Maybe adding something like `empty` filter might make it easier to check values for similar cases?

**Comment by zekker6** [comment]
@Z9n2JktHlZDmlhSvqc9X2MmL3BwQG7tk Starting from https://github.com/VictoriaMetrics/VictoriaMetrics/commit/4a49577028bd3b8def52d53ee2f75992999bb768 vmalert should convert `nil` to an empty string similar to Prometheus.
This commit will be a part of the next release. 

In the meantime, could you please try the following image and check if it helps?
```
docker.io/victoriametrics/vmalert:heads-oss-master-0-g4a4957702
```
If you don't use docker for deployment you can build a binary by using this [doc](https://github.com/VictoriaMetrics/VictoriaMetrics#how-to-build-from-sources) and check if that works for you.

**Comment by Z9n2JktHlZDmlhSvqc9X2MmL3BwQG7tk** [comment]
Thanks ! I will wait for the next release.

**Comment by valyala** [comment]
FYI, the issues should be fixed in [vmalert v1.90.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
